sql/codegen.go

```diff
@@ -193,18 +193,16 @@ try:
 except:
 	pass
 
-from sqlflow_submitter.db import connect, execute, insert_values
+from sqlflow_submitter.db import connect, execute, insert_values, db_generator, db_generator_predict
```

Comment:
The `test.sh` script runs in the container of the latest `sqlflow` image, we need to commit the update of `db.py` firstly in another PR before using the newly added API, otherwise `CI` will fail.

Reply:
I think `db.py` could be covered by CI because the PR's code branch is mounted under `$GOPATH` and we always import from `$GOPATH` in the dockerfile env settings.

Reply:
You are right, this is not a problem.